src/components/PartnerStatsCard/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import Card from '../Dashboard/Card';
+import './partnersStatsCard.scss';
+
+class PartnerStats extends Component {
+  renderStatsCard = (type) => {
+    const renderDiv = value => (
+      <div className="stats-value">{value}</div>
+    );
+    const statsCard = {
+      newPartner: renderDiv(7),
+      partnerChurns: renderDiv(2),
```

Comment:
@mikenthiwa good work on matching the mock ups. But quick question is this data expected to be static?

Reply:
We will read data from props, currently, there is no backend data that is available for this feature.